src/Service/ImageManager.php

```diff
@@ -46,6 +48,10 @@ public static function isImageType(string $mediaType): bool
         return \in_array($mediaType, self::IMAGE_MIMETYPES);
     }
 
+    /**
+     * @throws \Exception                     if the file could not be found
+     * @throws ImageDownloadTooLargeException
```

Comment:
I think you don't throw `ImageDownloadTooLargeException` this anymore here..

Reply:
true